api-reference/authentication.md

```diff
@@ -2,4 +2,4 @@
 
 The Platform API supports both signed out and authenticated modes. When users are in the signed-out state, only the API key is required. You can authenticate as a user by sending a 'X-Typekit-Token' HTTP header.
```

Comment:
Hmm.. If I'm pretty sure this is inaccurate. `x-typekit-token` was the auth header for our legacy v1 API, and is not a thing for Platform. 

To authenticate a user in Platform, we use the standard OAuth 2.0 header: `Authentication: Bearer {user_token}`. To pass in the API key, you add header `X-Api-Key: {Client ID}`. The latter is required always, the former only to call APIs in signed-in mode.

Reply:
I'll open a separate pull to address this.

Reply:
https://github.com/typekit/fonts-api-docs/pull/2